qiita_ware/util.py

```diff
@@ -108,113 +107,24 @@ def stats_from_df(df):
 
     Returns
     -------
-    dict
-        Dictionary object where the keys are the names of the metadata
-        categories and the keys are tuples where the first element is the name
+    dict of list of tuples
+        Dictionary object where the keys are the metadata categories
+        and the values are list of tuples where the first element is the name
         of a metadata value in category and the second element is the number of
         times that value was seen.
+        Format {category: [(val1, count1), (val2, count2), ...], ...}
     """
     out = {}
 
     for column in natsorted(df.columns):
+        if column == 'study_id':
```

Comment:
Wouldn't it be easier to remove the column 'study_id' and then iterate through all the columns without having to check for every iteration?